composer.json

```diff
@@ -83,7 +83,7 @@
         "allow-plugins": {
             "infection/extension-installer": true,
             "composer/package-versions-deprecated": true,
-            "yiisoft/config": true
+            "yiisoft/config": false
```

Comment:
Why?

Reply:
Composer plugin is not used here. It works on app level only. Here Yii Config needs for tests only.